pkg/gui/filetree/file_manager.go

```diff
@@ -35,6 +45,35 @@ func (m *FileManager) ExpandToPath(path string) {
 	m.collapsedPaths.ExpandToPath(path)
 }
 
+func (m *FileManager) GetFilesForDisplay() []*models.File {
```

Comment:
I don't to what extent you want this tested?

Reply:
It would be good to have a test for this i.e. three test cases: one with DisplayAll, one with DisplayStaged, and one with DisplayUnstaged